### PR TITLE
Create LICENSE.md and updated files to use NPM modules from THE APOLLOS PROJECT

### PR DESCRIPTION
**Pull request by IsaacHardy**
## DESCRIPTION
Creates LICENSE.md and updates files to use NPM modules from THE APOLLOS PROJECT
### What does this PR do, or why is it needed?
NPM requires we have license setup for our modules before publishing them.
### What design trade-offs/decisions were made?
N.A.
### How do I test this PR?
N.A.
### What automated tests did you write?
N.A.
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#### THIS LICENSE PERMITS THE USE, REPRODUCTION, PREPARATION OF DERIVATIVE WORKS OF, PUBLIC DISPLAY, PUBLIC PERFORMANCE, SUBLICENSE, AND DISTRIBUTION OF THE WORK AND DERIVATIVE WORKS SOLELY FOR INTERNAL USE BY FAITH BASED ORGANIZATIONS. FOR ANY USE, REPRODUCTION, PREPARATION OF DERIVATIVE WORKS OF, PUBLIC DISPLAY, PUBLIC PERFORMANCE, SUBLICENSE, AND DISTRIBUTION OF THE WORK AND DERIVATIVE WORKS FOR ANY OTHER PURPOSE, PLEASE CONTACT SPARK DEVELOPMENT, INC. FOR A COMMERCIAL LICENSE AT info@sparkdevnetwork.org
```
Comment (conrad-vanl) [inline LICENSE.md]: We need to change this language not to include Spark and spark's email :)

Comment (IsaacHardy) [reply LICENSE.md]: I'm waiting on Michael to give me a better email then what I've updated it to, but for now I've changed it to `web@newspring.cc`

**Comment by conrad-vanl** [comment]
@vinnyjth builds should be fixed now.

The one thing holding this up is I'd like to look into `apolloschurchapp` some more and see if I can further improve its metro+babel+yarn workspace setup any more, but I'm sure this PR can be merged as is now.

**Comment by conrad-vanl** [comment]
Oh, @vinnyjth I did confirm that it _is_ symlinking the workspace packages now correctly.

**Comment by conrad-vanl** [comment]
Actually, we need to hold off on this. app builds failing now, which is actually okay - it's behaving like I'd expect it to versus the weirdness we've been having. let me try to clean it up.

**Comment by conrad-vanl** [comment]
Closed in favor of #431
